lib/ansible/modules/system/aix_lvg.py

```diff
@@ -0,0 +1,404 @@
+
+        changed = True
+        if not module.check_mode:
+            mkvg_cmd = module.get_bin_path("mkvg", True)
+            rc, _, err = module.run_command(
+                "%s %s %s %s -y %s %s" % (
+                    mkvg_cmd, vg_opt[vg_type], pp_size, force_opt[force],
+                    vg, ' '.join(pvs)))
+
+            if rc == 0:
+                module.exit_json(
+                    changed=True, msg="Volume group %s created." % vg)
+            else:
+                module.exit_json(
+                    changed=False, msg="Creating volume group %s failed." % vg,
+                    rc=rc, err=err)
+
+        return changed
+
+
+def reduce_vg(module, vg, pvs, vg_validation):
+    vg_state, msg = vg_validation
+
+    if vg_state is False:
+        module.exit_json(
+            changed=False, msg=msg)
+
+    # Define pvs_to_remove (list of physical volumes to be removed).
+    if pvs is None:
+        # Remove VG if pvs are note informed.
+        # Remark: AIX will permit remove only if the VG has not LVs.
+        lsvg_cmd = module.get_bin_path("lsvg", True)
+        rc, current_pvs, err = module.run_command("%s -p %s" % (lsvg_cmd, vg))
+        if rc != 0:
+            module.fail_json(msg="Failing to execute %s command." % lsvg_cmd)
+
+        pvs_to_remove = []
+        for line in current_pvs.splitlines()[2:]:
+            pvs_to_remove.append(line.split()[0])
+
+        reduce_msg = "Volume group %s removed." % vg
+    else:
+        pvs_to_remove = pvs
+        reduce_msg = (
+            "Physical volume(s) %s removed from Volume group %s." % (
+                " ".join(pvs_to_remove), vg))
+
+    # Reduce volume group.
+    if len(pvs_to_remove) > 0:
+        changed = True
+
+        if module.check_mode:
+            reducevg_cmd = module.get_bin_path("reducevg", True)
+            rc, _, err = module.run_command(
+                "%s -df %s %s" % (reducevg_cmd, vg, " ".join(pvs_to_remove)))
+
+            if rc == 0:
+                module.exit_json(changed=True, msg=reduce_msg)
+            else:
+                module.fail_json(
+                    msg="Unable to remove %s." % vg, rc=rc, err=err)
+
+        return changed
+
+
+def state_vg(module, vg, state, vg_validation):
+    vg_state, msg = vg_validation
+
+    if vg_state is None:
+        module.fail_json(msg=msg)
+
+    if state == 'varyon':
+        if vg_state is False:
+            changed = True
+
+            if not module.check_mode:
+                varyonvg_cmd = module.get_bin_path("varyonvg", True)
+                rc, varyonvg_out, err = module.run_command(
+                    "%s %s" % (varyonvg_cmd, vg))
+                if rc != 0:
+                    module.fail_json(msg="Varyonvg failed.", rc=rc, err=err)
+                else:
+                    module.exit_json(
+                        changed=True,
+                        msg="Varyon volume group %s completed." % vg)
+
+            return changed
```

Comment:
Why do you return something when you are not doing nothing with it in the `main()` function? What about to use `module.exit_json()` only in the `main()` function? The same in other functions.

Reply:
Could I do it as I did in the comment above?